frontend/src/scenes/feature-flags/FeatureFlagReleaseConditionsLogic.ts

```diff
@@ -183,9 +183,20 @@ export const featureFlagReleaseConditionsLogic = kea<featureFlagReleaseCondition
                 actions.setAffectedUsers(index, undefined)
 
                 const properties = condition.properties
-                if (!properties || properties?.length === 0 || properties.some(isEmptyProperty)) {
-                    // don't compute for full rollouts or empty conditions
+                if (!properties || properties.some(isEmptyProperty)) {
+                    // don't compute for incomplete conditions
                     usersAffected.push(Promise.resolve({ users_affected: -1, total_users: -1 }))
+                } else if (properties.length === 0) {
+                    // Request total users for empty condition sets
+                    const responsePromise = api.create(
+                        `api/projects/${values.currentTeamId}/feature_flags/user_blast_radius`,
+                        {
```

Comment:
yeah, you probably know this too, but if we don't have any properties in this request we just pull a value off the `Team` object on the backend and use that, rather than querying Postgres again.  Seems like a safe function to reuse.  I like it!

Reply:
Yep I did take a look at how expensive the extra queries were, not too bad considering!